config/migrations/2023/20231221132600-fix-duplicate-lang-strings.sparql

```diff
@@ -0,0 +1,16 @@
+DELETE
+{
+  GRAPH ?g {
+    ?s ?p ?oLang.
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?s ?p ?oLang.
+    ?a ?b ?o .
```

Comment:
Shouldn't the subject and property match as well?

i.e.:
```
?s ?p
  ?oLang,
  ?o.
```

Reply:
You are right, I based it on a query made by Claire and didn't realise that problem, I fixed it now